electrum/constants.py

```diff
@@ -39,136 +41,241 @@ def read_json(filename, default):
         r = default
     return r
 
-
-GIT_REPO_URL = "https://github.com/spesmilo/electrum"
-GIT_REPO_ISSUES_URL = "https://github.com/spesmilo/electrum/issues"
+GIT_REPO_URL = "https://github.com/BTCGPU/electrum"
+GIT_ISSUE_URL = GIT_REPO_URL + "/issues"
```

Comment:
```suggestion
GIT_REPO_ISSUES_URL = GIT_REPO_URL + "/issues"
```

And change the other files as well. This will be good for us to keep sync with the upstream